js/window-shade.js

```diff
@@ -0,0 +1,20 @@
+namespacer('bcpl');
+
+bcpl.windowShade = (($) => {
+	const windowShadeSelector = '#window-shade';
+	let timeout;
+
+	const cycle = (displaySpeed, delaySpeed) => {
+		const $windowShade = $(windowShadeSelector);
+		clearTimeout(timeout);
+		$windowShade.slideDown(displaySpeed, () => {
```

Comment:
move to it's own function?

Reply:
The problem is that I'd have to send params into setTimeout beyond the function and the expiration, which isn't supported in IE9 (we still support it).